src/WCSimEventAction.cc

```diff
@@ -626,7 +629,7 @@ void WCSimEventAction::EndOfEventAction(const G4Event* evt)
   }
 
   //fill correct variables for track from decay
-   G4cout << " Filling Root Event " << G4endl;
+   //G4cout << " Filling Root Event " << G4endl;
```

Comment:
Ya, just go through and remove these changes you made that aren't part of the radioactivity simulation.